Src/Plugins/Security/SEC101.SecurePlaintextSecrets.json

```diff
@@ -80,7 +80,7 @@
         {
           "Id": "SEC101/016",
           "Name": "DoNotExposePlaintextSecrets/StripeApiKey",
-          "ContentsRegex": "(?:r|s)k_[live|test]_(?i)[0-9a-z]{24}",
```

Comment:
>[live|test] [](start = 37, length = 11)

Look at this bogus pattern! should have been (live|test), was rendered as [live|test]